src/bundle/Command/CronRunCommand.php

```diff
@@ -13,9 +13,24 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Psr\Log\LoggerInterface;
 
 class CronRunCommand extends ContainerAwareCommand
 {
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @injectService $logger @logger
+     */
```

Comment:
@homarx what is the purpose of this annotation?

Reply:
@alongosz To be honest: I took it over from vendor/ezsystems/ezplatform-admin-ui/src/lib/Behat/Helper/Hooks.php  ; )
Should I remove it?

Reply:
> @alongosz To be honest: I took it over from vendor/ezsystems/ezplatform-admin-ui/src/lib/Behat/Helper/Hooks.php ; )
> Should I remove it?

Well... Yes. This command is not a Behat service. Such Behat notation no longer works anyway in Symfony 5.
I missed this one. You can skip this PHP Doc for the constructor entirely because of strict types already present there.

Reply:
@alongosz I removed the annotation